### PR TITLE
MEN-4178 help & docs reference adjustments

### PR DESCRIPTION
**Pull request by mzedel**

### CODE REVIEW

File: src/js/components/common/dialogs/__snapshots__/virtualdeviceonboarding.test.js.snap [hunk 1]
```diff
@@ -83,12 +83,11 @@ exports[`VirtualDeviceOnboarding Component renders correctly 1`] = `
   <p>
     Visit the 
     <a
-      href="/help/application-updates/demo-virtual-device"
-      onClick={[Function]}
+      href="https://docs.mender.io/development/get-started/preparation/prepare-a-virtual-device"
```
Comment (michaelatmender) [inline src/js/components/common/dialogs/__snapshots__/virtualdeviceonboarding.test.js.snap:86]: I'd make this target="_blank" too

File: src/js/components/common/dialogs/__snapshots__/deviceconnectiondialog.test.js.snap [hunk 2]
```diff
@@ -76,7 +76,16 @@ exports[`DeviceConnectionDialog Component renders correctly 1`] = `
                 id="deb-package-tip"
               >
                 <p>
-                  The steps in the guide should work on most operating systems in the debian family (e.g. Debian, Ubuntu, Raspberry Pi OS) and devices based on ARMv6 or newer (e.g. Raspberry Pi 2/3/4, Beaglebone).
+                  The steps in the guide should work on most operating systems in the debian family (e.g. Debian, Ubuntu, Raspberry Pi OS) and devices based on ARMv6 or newer (e.g. Raspberry Pi 2/3/4, Beaglebone). Visit
```
Comment (michaelatmender) [inline src/js/components/common/dialogs/__snapshots__/deviceconnectiondialog.test.js.snap:79]: 'debian family' should be 'Debian family', it's been the incorrect case for a while!

**Review by michaelatmender — CHANGES_REQUESTED** [review]
Just two tiny changes requested, otherwise looks good!

**Comment by estenberg** [comment]
Great improvement!

It is a bit hard to visualize how it will look based on the code but the changes look good to me and can take a look at the Help section once merged to see if there are any minor changes I'd suggest at that point too :+1:

**Review by estenberg — APPROVED** [review]

**Review by michaelatmender — APPROVED** [review]

**Pull request merged**